ISLE/emscripten/config.cpp

```diff
@@ -0,0 +1,17 @@
+#include "config.h"
+
+#include "filesystem.h"
+
+#include <SDL3/SDL_log.h>
+#include <iniparser.h>
+
+void Emscripten_SetupDefaultConfigOverrides(dictionary* p_dictionary)
+{
+	SDL_Log("Overriding default config for Emscripten");
+
+	iniparser_set(p_dictionary, "isle:diskpath", Emscripten_bundledPath);
+	iniparser_set(p_dictionary, "isle:cdpath", Emscripten_streamPath);
+	iniparser_set(p_dictionary, "isle:savepath", Emscripten_savePath);
+	iniparser_set(p_dictionary, "isle:Full Screen", "false");
+	iniparser_set(p_dictionary, "isle:Flip Surfaces", "true");
```

Comment:
Doesn't the game switch off flip surface if full screen is false?

Reply:
I think a lot of code is never actually called, like `MxVideoManager::Create` and subsequently `MxDisplaySurface::Create`

Reply:
> I think a lot of code is never actually called

Yeah I think this game has potential for some major dead code removal :D